plugin/rufo.vim

```diff
@@ -12,13 +16,22 @@ if !exists('g:rufo_errors_buffer_position')
   let g:rufo_errors_buffer_position = 'bottom'
 endif
 
+if get(g:, "rufo_saving", 0)
+  autocmd BufWritePre *.rb silent! call rufo#format_file()
```

Comment:
@lucerion Thank you for the review.I forgot to delete the draft code.
Firstable I wrote it and then updated to `RufoFormatBeforeSave`.
I will delete this line.